src/views/homepage/components/hero/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { ReactElement } from 'react'
+import Link from 'next/link'
+import classNames from 'classnames'
+import Badge from 'components/badge'
+import Heading from 'components/heading'
+import Text from 'components/text'
+import s from './hero.module.css'
+import slugify from 'slugify'
+
+type ActionProps = {
+  heading: string
+  description: string
+  link: string
+  linkText: string
+  theme?: 'green' | 'purple'
```

Comment:
The thinking here is that in the future this could include product slugs that have there branding gradient.